app/controllers/sessions_controller.rb

```diff
@@ -1,8 +1,12 @@
 class SessionsController < Clearance::SessionsController
 
   def create
-    if params[:session][:password] && User.find_by(email: params[:session][:email]).encrypted_password == 'x'
-      redirect_to  new_password_path, flash: { :info => "Due to a security update from our side we kindly ask you to reset your password." }
+    if User.find_by(email:params[:session][:email]).nil?
+      redirect_to sign_up_path, flash: { :alert => "Account not found, please sign up"}
```

Comment:
maybe: "Account not found, please register first" (i think it makes clearer whats happening, so people dont confuse sign up with sign in)

Reply:
what happens to the case if someone enters no password?